keepercommander/api.py

```diff
@@ -5,7 +5,7 @@
 #              |_|            
 #
 # Keeper Commander 
-# Copyright 2017 Keeper Security Inc.
+# Copyright 2021 Keeper Security Inc.
```

Comment:
FWIW, ISTR that a company should update their copyright notices for appropriate years, annually.

Reply:
Commander is the first and only public source code repository. There are no established rules on updating copyright.
Developers usually modify the header when they touch the file

Reply:
I personally find docstrings and comments distracting.
Type hints and lack of IDE warnings are more helpful for me.